Model/Api/PaymentRequest.php

```diff
@@ -88,6 +88,12 @@ private function createClient($storeId) {
         $client->setUsername($webserviceUsername);
         $client->setPassword($webservicePassword);
 
+        $client->setAdyenPaymentSource($this->_adyenHelper->getModuleName(), $this->_adyenHelper->getModuleVersion());
+
+		$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+		$productMetadata = $objectManager->get('Magento\Framework\App\ProductMetadataInterface');
+		$client->setExternalPlatform($productMetadata->getName(), $productMetadata->getVersion());
```

Comment:
Let's not use objectManager, best to use dependency injection in the constructor.